packages/node_modules/@ciscospark/widget-demo/src/components/demo-widget/index.js

```diff
@@ -134,6 +141,13 @@ class DemoWidget extends Component {
     return this.setState({mode: value, destinationId: ''});
   }
 
+  @autobind
+  handleActivitiesChange(event, isInputChecked) {
+    const activities = {...this.state.activities};
```

Comment:
This can be written more succinctly like this: `const {activities} = this.state;`

Reply:
Got it, thanks!